recitation-8/words.json

```diff
@@ -13231,7 +13231,6 @@
     "animists": 1,
     "animize": 1,
     "animized": 1,
-    "animo": 1,
```

Comment:
hmmm, this line seems to be a problem.

Reply:
please change the word list to add this word back in.

Reply:
@MichaelHilton I think the problem is the test. Animo is a stupid word.

Reply:
Need to add word back into dictionary for the test.

Reply:
You took out this word "animo".